src/Foundation/Enum.cs

```diff
@@ -1258,4 +1259,51 @@ public enum NSMeasurementFormatterUnitOptions : nuint {
 		TemperatureWithoutUnit = (1 << 2)
 	}
 
+	[Watch(2, 0), TV(9, 0), Mac(10, 8), iOS(6, 0)]
+	[Native]
+	public enum NSXPCConnectionOptions : nuint
```

Comment:
Realized this one after I pushed. I have all the Xpcs changed locally for when I push again